src/content/configuration/resolve.md

```diff
@@ -452,6 +453,23 @@ module.exports = {
 };
 ```
 
+### `resolve.restrictions`
+
+`[string | RegExp]`
```

Comment:
https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`[string, RegExp]`
```